src/libAtomVM/mailbox.h

```diff
@@ -64,18 +64,21 @@ enum MessageType
     TrapExceptionSignal,
 };
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
what kind of warning are we trying to ignore? is that one about access outside struct bounds?

Reply:
I believe it is.

Reply:
This was removed after rebasing on top of the work on mailbox message structs.
Please note that the new structs are a little bit different, so a word or two are wasted now when a message is converted to a heap fragment.